Source/Core/DistanceDisplayCondition.js

```diff
@@ -0,0 +1,136 @@
+/*global define*/
+define([
+        './Cartesian3',
+        './defaultValue',
+        './defined',
+        './defineProperties',
+        './Matrix4'
+    ], function(
+        Cartesian3,
+        defaultValue,
+        defined,
+        defineProperties,
+        Matrix4) {
+    'use strict';
+
+    /**
+     * Determines visibility based on the distance to the camera.
+     *
+     * @alias DistanceDisplayCondition
+     * @constructor
+     *
+     * @param {Number} [near=0.0] The smallest distance in the interval where the object is visible.
+     * @param {Number} [far=Number.MAX_VALUE] The largest distance in the interval where the object is visible.
+     *
+     * @example
+     * // Make a billboard that is only visible when the distance to the camera is between 10 and 20 meters.
+     * billboard.distanceDisplayCondition = new DistanceDisplayCondition(10.0 20.0);
+     */
+    function DistanceDisplayCondition(near, far) {
+        near = defaultValue(near, 0.0);
+        this._near = near;
+        this._near2 = near * near;
+
+        far = defaultValue(far, Number.MAX_VALUE);
+        this._far = far;
+        this._far2 = far * far;
+    }
+
+    defineProperties(DistanceDisplayCondition.prototype, {
+        /**
+         * The smallest distance in the interval where the object is visible.
+         * @memberof DistanceDisplayCondition.prototype
+         * @type {Number}
+         * @default 0.0
+         */
+        near : {
+            get : function() {
+                return this._near;
+            },
+            set : function(value) {
+                this._near = value;
+                this._near2 = value * value;
```

Comment:
Nitpicky, but as a generate practice, we would normally call this `nearSquared`.